packages/devtools_app/lib/src/shared/console/widgets/evaluate.dart

```diff
@@ -414,12 +414,19 @@ class ExpressionEvalFieldState extends State<ExpressionEvalField>
 
   /// If [expressionText] is assignment like `var x=$1`, processes it.
   ///
-  /// Returns true is the text was parsed as assignment.
+  /// Returns true if the text was parsed as assignment.
   bool _tryProcessAssignment(String expressionText) {
     if (!FeatureFlags.evalAndBrowse) return false;
 
     final assignment = ConsoleVariableAssignment.tryParse(expressionText);
     if (assignment == null) return false;
+    const result = true;
```

Comment:
Nit: if we're going to use a constant for the success case, let's just name this constant to represent that. Maybe something like `const kSuccess = true`?